src/entities/hotBoard/ui/HotBoardListRow.tsx

```diff
@@ -22,46 +51,91 @@ interface HotBoardListRowProps {
  */
 export default function HotBoardListRow({
   boardId,
-  rank,
+  rank = 0,
   keyword,
   count,
   views,
   timer,
 }: HotBoardListRowProps) {
+  const isMedal = rank >= 1 && rank <= 3;
   return (
-    <Link href={`/hotboard/${boardId}`}>
-      <div className="flex cursor-pointer items-center justify-between rounded-2xl border border-gray-200 p-3 hover:bg-gray-100 md:rounded-[20px] md:p-4">
-        <span className="flex min-w-0 items-center gap-x-3">
-          <span className="flex h-7 w-7 items-center justify-center text-base font-semiBold text-gray-800 md:text-xl">
-            {rank}
-          </span>
-          <span className="truncate text-md font-semiBold text-gray-800 md:text-lg">{keyword}</span>
+    <Link
+      href={`/hotboard/${boardId}`}
+      className={cn(
+        'flex h-[60px] cursor-pointer items-center justify-between rounded-2xl px-3 md:rounded-[20px] md:px-4',
+        isMedal
+          ? 'bg-brand-100 hover:bg-[#EDF5FF] md:h-[68px]'
+          : 'border border-gray-200 hover:bg-gray-100'
+      )}
+    >
+      {/* 좌측: 순위 및 키워드 */}
+      <div
+        className={cn(
+          'flex min-w-0 flex-1 items-center',
+          isMedal ? 'gap-2 md:gap-x-4' : 'gap-1 md:gap-x-3'
+        )}
+      >
+        {rank > 0 && <RankBadge rank={rank} />}
+        <span
+          className={cn(
+            'truncate',
+            isMedal
+              ? 'text-base font-bold text-brand-500 md:text-xl'
+              : 'text-md font-semibold text-gray-800 md:text-lg'
+          )}
+        >
+          {keyword}
         </span>
-        <span className="flex flex-col items-end gap-x-2 md:flex-row md:items-center">
-          <span className="hidden w-16 text-center text-md font-regular text-gray-500 md:block">
+      </div>
+
+      {/* 우측: 통계 및 타이머 */}
+      <div className="flex flex-col-reverse gap-x-2 md:flex-row">
+        <div className="flex justify-end gap-x-2 text-2xs font-regular text-gray-500 md:text-md">
+          <div className='flex w-auto items-center justify-center gap-1 after:ml-2 after:h-1.5 after:w-px after:bg-gray-300 after:content-[""] md:w-16 md:after:content-none'>
+            <WriteIcon className="block md:hidden" />
```

Comment:
```suggestion
            <Pencil className="h-4 w-4 text-gray-500 md:hidden" />
```
기존에 있던 Pencil이라는 아이콘과 동일해 보이는데 위처럼 사용해주실 수 있을까요?
(이름이 마음에 안 드시면 바꾸셔도 됩니다)

Reply:
기존의 Pencil을 PencilIcon으로 이름 수정후, 통일시켰습니다